### PR TITLE
Fixes for Job Run History Page

### PR DESCRIPTION
**Pull request by GaneshSPatil**
Description:

* Change page header from 'Agent Job Run History for AGENT_UUID' to
  'Agent Job Run History' as AGENT_UUID is internal no additional
  information about agent is available.

* Render help tooltips for wait, building and total time.

Screen Shot 2019-11-28 at 6 00 40 PM

### CODE REVIEW

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx [hunk 1]
```diff
@@ -37,14 +37,43 @@ export class AgentJobStateTransitionModal extends Modal {
...
+          <div class={styles.key} data-test-id="key-job">Job</div>
+          <span class={styles.value} data-test-id="value-job">
+            : {this.jobRepresentation()}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-wait-time">
+            Wait Time
+            <Tooltip.Info content={"Wait Time is the time spent by the job waiting for an agent."}/>
```
Comment (GaneshSPatil) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx]: @adityasood @maheshp -- does this description sound right? is there a better way to phrase this?

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx [hunk 2]
```diff
@@ -37,14 +37,43 @@ export class AgentJobStateTransitionModal extends Modal {
...
+          </div>
+          <span class={styles.value} data-test-id="value-wait-time">
+            : {this.findTimeDifference("Scheduled", "Assigned")}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-building-time">
+            Building Time
+            <Tooltip.Info content={"Building time is the time spent building the job on an agent."}/>
```
Comment (GaneshSPatil) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx]: @adityasood @maheshp -- does this description sound right? is there a better way to phrase this?

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx [hunk 3]
```diff
@@ -37,14 +37,43 @@ export class AgentJobStateTransitionModal extends Modal {
...
+          </div>
+          <span class={styles.value} data-test-id="value-building-time">
+            : {this.findTimeDifference("Assigned", "Completed")}
+          </span>
+        </div>
+
+        <div data-test-id="key-value-pair" class={styles.keyValuePair}>
+          <div class={styles.key} data-test-id="key-total-time">
+            Total Time
+            <Tooltip.Info content={"Total time is the time taken by the job from scheduling to completion. This is the total time spent by the job in waiting and building state."}/>
```
Comment (GaneshSPatil) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx]: @adityasood @maheshp -- does this description sound right? is there a better way to phrase this?

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx (hunk 1 above)
Comment (adityasood) [reply server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx]: Time spent by the job waiting for an agent to be assigned?

Comment (GaneshSPatil) [reply server/src/main/webapp/WEB-INF/rails/webpack/views/pages/agent-job-run-history/agent_job_state_transitions_modal.tsx]: @adityasood -- Thanks, changed accordingly!!

**Pull request merged**